src/pools/selection-strategies.ts

```diff
@@ -3,12 +3,23 @@ import type { IWorker } from './abstract-pool'
 import type { IPoolInternal } from './pool-internal'
 
 /**
- * Worker choice strategies string list.
+ * Enumeration of worker choice strategies.
  */
-export enum WorkerChoiceStrategy {
-  ROUND_ROBIN = 'RoundRobin',
-  DYNAMIC = 'Dynamic'
-}
+export const WorkerChoiceStrategies = Object.freeze({
+  /**
+   * TODO: Add description.
+   */
+  ROUND_ROBIN: 'ROUND_ROBIN',
+  /**
+   * TODO: Add description.
```

Comment:
@jerome-benoit Do you have a good description for this one?

Reply:
Round robin worker selection strategy ;)